# A set of collision operations that will be performed in the order they are specified
CollisionOperation[] collision_operations